bpf/lib/common.h

```diff
@@ -287,6 +287,9 @@ enum {
  */
 #define IPSEC_ENDPOINT TC_ACT_OK
 
+/* Indicates that proxy redirection is required */
+#define TC_ACT_PROXY_REDIRECT (1 << 16)
```

Comment:
`TC_ACT_*` suggests that this is an action that TC understands. This is not a TC command.

```suggestion
#define POLICY_ACT_PROXY_REDIRECT (1 << 16)
```